composer.json

```diff
@@ -17,19 +17,20 @@
         "php": "^5.5 || ^7.0",
         "cboden/ratchet": "0.3.*",
         "doctrine/orm": "2.5.*",
-        "doctrine/phpcr-bundle": "~1.2.4",
+        "doctrine/phpcr-bundle": "^1.2.4",
         "doctrine/doctrine-bundle": "~1.0",
         "doctrine/doctrine-cache-bundle": "~1.0",
-        "friendsofsymfony/http-cache": "1.3.*",
+        "friendsofsymfony/http-cache": "~1.3",
         "friendsofsymfony/rest-bundle": "~1.4",
         "guzzlehttp/guzzle": "^6.2",
         "imagine/imagine": "~0.6.1",
         "massive/search-bundle": "0.16.*",
         "sulu/document-manager": "dev-develop",
         "sensio/framework-extra-bundle": "3.0.*",
-        "stof/doctrine-extensions-bundle": "1.1.*",
+        "stof/doctrine-extensions-bundle": "~1.1",
         "symfony-cmf/routing-bundle": "1.2.*",
         "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/security-acl": "~2.8",
```

Comment:
we need that for the update of symfony3 where this component is a own dependency. but we should not introduce it until we have symfony3 as dependency. i will remove it.